From 03083650e4aa326b86d0f12a08962f56cf618f44 Mon Sep 17 00:00:00 2001
From: SubhadeepJasu <subhadeep107@proton.me>
Date: Sun, 4 Jan 2026 11:23:22 +0530
Subject: [PATCH] Remove extra blueprint-compile command

---
 data/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index 7e4e1e0..9d6120f 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -45,7 +45,6 @@ foreach b : blueprint_files
     bfs += meson.current_source_dir () / b
 endforeach

-run_command ('blueprint-compiler', 'batch-compile', meson.project_build_root () / 'data/ui', meson.current_source_dir () / 'ui', bfs, check: true)

 blueprints = custom_target ('blueprints',
     input: files (blueprint_files),
--
2.43.0

